analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/FileMetadataAnalyzerTest.cs

```diff
@@ -85,12 +85,12 @@ public void NotAutogenerated(ProjectType projectType)
         {
             var notAutogeneratedFiles = new[]
             {
-                @"normal_file.cs",
-                @"generated_region.cs",
-                @"generated_region_2.cs"
+                @$"{BasePath}normal_file.cs",
+                @$"{BasePath}generated_region.cs",
+                @$"{BasePath}generated_region_2.cs"
             };
 
-            var projectFiles = notAutogeneratedFiles.Select(x => BasePath + x).ToArray();
+            var projectFiles = notAutogeneratedFiles;
```

Comment:
This rename doesn't make sense anymore. Let's remove `projectFiles` variable and use `notAutogeneratedFiles` instead (because that one is more descriptive)